### PR TITLE
ENH JTFS `create_filters`

### PR DESCRIPTION
**Pull request by lostanlen**
segue of #927 and #928

thanks to #842 #915 #922 , there is no need for a dedicated `jtfs_filter_factory`. I managed to get around with calling the `scattering_filter_factory` multiple times:
1. for time scattering 
```
super(TimeFrequencyScatteringBase, self).create_filters()
```

2. for `average_local_fr`
```python
phi_fr_f, = scattering_filter_factory(self._N_padded_fr, self.J_fr, (), self.F, self.filterbank_fr)
```
            
3. for actual frequency scattering (WIP)
```
phi1_fr_f, psis_fr_f = scattering_filter_factory(self._N_padded_fr, self.J_fr, self.Q_fr, 2**self.J_fr, self.filterbank_fr)
```

Note the use of `spin` in the `filterbank_fr` property:
```python
@property
def filterbank_fr(self):
    filterbank_kwargs = {
        "alpha": self.alpha, "r_psi": self.r_psi, "sigma0": self.sigma0}
    return spin(anden_generator, filterbank_kwargs)
```

The 1D module remains unaffected! No `is_spinned` boolean in the namespace ;)

I don't imagine i'm introducing any missed lines in the test coverage but feel free to propose ideas for testing `filters_fr`

### CODE REVIEW

File: kymatio/scattering1d/frontend/base_frontend.py [hunk 1]
```diff
@@ -539,4 +522,70 @@ def _document(cls):
             n=cls._doc_array_n)
 
 
-__all__ = ['ScatteringBase1D']
+class TimeFrequencyScatteringBase(ScatteringBase1D):
+    def __init__(self, *, J, J_fr, shape, Q, T=None, oversampling=0,
```
Comment (cyrusvahidi) [inline kymatio/scattering1d/frontend/base_frontend.py:526]: can you make this `*args`

Comment (lostanlen) [reply kymatio/scattering1d/frontend/base_frontend.py:526]: https://peps.python.org/pep-3102/
keyword-only arguments

**Review by cyrusvahidi — COMMENTED** [review]
i've reviewed this and all seems good. Let's just add tests like we had in #803?

**Comment by cyrusvahidi** [comment]
let's add the filterbank tests in the next PR

**Review by cyrusvahidi — APPROVED** [review]

**Pull request merged**